docs/config.md

```diff
@@ -1,211 +1,466 @@
+   optional.
+
+If a combination of these three methods is used then Element will fail to load. This is because it is unclear which
+should be considered "first".
+
+## Labs flags
+
+Labs flags are optional, typically beta or in-development, features that can be turned on or off. The full range of
+labs flags and their development status are documented [here](./labs.md). If interested, the feature flag process is
+documented [here](./feature-flags.md).
+
+To force a labs flag on or off, use the following:
+
+```json
+{
+   "features": {
+      "feature_you_want_to_turn_on": true,
+      "feature_you_want_to_keep_off": false
+   }
+}
+```
+
+If you'd like the user to be able to self-select which labs flags they can turn on, add `"show_labs_flags": true` to
+your config. This will turn on the tab in user settings.
+
+**Note**: Feature support varries release-by-release. Check the [labs flag documentation](./labs.md) frequently if enabling
+the functionality.
+
+## Default settings
+
+Some settings additionally support being specified at the config level to affect the user experience of your Element Web
+instance. As of writing those settings are not fully documented, however a few are:
+
+1. `default_federate`: When `true` (default), rooms will be marked as "federatable" during creation. Typically this setting
+   shouldn't be used as the federation capabilities of a room **cannot** be changed after the room is created.
+2. `default_country_code`: An optional ISO 3166 alpha2 country code (eg: `GB`, the default) to use when showing phone number
+   inputs.
+3. `room_directory`: Optionally defines how the room directory component behaves. Currently only a single property, `servers`
+   is supported to add additional servers to the dropdown. For example:
+   ```json
+   {
+      "room_directory": {
+         "servers": ["matrix.org", "example.org"]
+      }
+   }
+   ```
+4. `setting_defaults`: Optional configuration for settings which are not described by this document and support the `config`
+   level. This list is incomplete. For example:
+   ```json
+   {
+      "setting_defaults": {
+         "MessageComposerInput.showStickersButton": false,
+         "MessageComposerInput.showPollsButton": false
+      }
+   }
+   ```
+   These values will take priority over the hardcoded defaults for the settings.
+
+## Customisation & branding
+
+<!-- Author's note: https://english.stackexchange.com/questions/570116/alternative-ways-of-saying-white-labeled -->
+
+Element supports some customisation of the user experience through various branding and theme options. While it doesn't support
+complete re-branding/private labeling, a more personalised experience can be achieved for your users.
+
+1. `default_theme`: Typically either `"light"` (the default) or `"dark"`, this is the optional name of the colour theme to use.
+   If using custom themes, this can be a theme name from that as well.
```

Comment:
```suggestion
1. `default_theme`: Typically either `light` (the default) or `dark`, this is the optional name of the colour theme to use.
   If using custom themes, this can be a theme name from that as well.
```